addons/project/static/src/views/project_task_form/project_task_form_controller.js

```diff
@@ -14,9 +16,39 @@ Are you sure you want to proceed?`
 );
 
 export class ProjectTaskFormController extends FormControllerWithHTMLExpander {
+    static template = "project.ProjectTaskFormView";
+    static components = {
+        ...FormControllerWithHTMLExpander.components,
+        ProjectTaskTemplateDropdown,
+    };
+
+    static props = {
+        ...FormControllerWithHTMLExpander.props,
+        focusTitle: {
+            type: Boolean,
+            optional: true,
+        },
+    };
+    static defaultProps = {
+        ...FormControllerWithHTMLExpander.defaultProps,
+        focusTitle: false,
+    };
+
     setup() {
         super.setup();
         this.notifications = useService("notification");
+
+        if (this.props.focusTitle) {
+            useEffect(
+                () => {
+                    if (this.rootRef) {
+                        const title = this.rootRef.el.querySelector("#name_0");
+                        title.focus();
```

Comment:
we should check if `title` is found to avoid having a traceback. 🙂 

```suggestion
                        const title = this.rootRef.el.querySelector("#name_0");
                        title && title.focus();
```